app/web/features/messages/messagelist/MessageList.tsx

```diff
@@ -35,34 +34,32 @@ export default function MessageList({
   className,
 }: MessageListProps) {
   const { t } = useTranslation(MESSAGES);
-  const classes = useStyles();
 
   return (
-    <div
-      className={classNames(classes.list, className)}
-      data-testid="message-list"
-    >
+    <List className={classNames(className)} data-testid="message-list">
```

Comment:
```suggestion
    <List className={className} data-testid="message-list">
```

Can just be `className` now since you put the `classes.list` in the styled component.